router/core/graphql_prehandler.go

```diff
@@ -597,6 +600,15 @@ func (h *PreHandler) handleOperation(req *http.Request, buf *bytes.Buffer, httpO
 
 	httpOperation.traceTimings.EndPlanning()
 
+	/**
+	* Validate that the planned query doesn't exceed the maximum query depth configured
+	 */
+
+	queryDepthErr := operationKit.ValidateQueryDepth(h.maxQueryDepth, opContext.preparedPlan.operationDocument, opContext.preparedPlan.schemaDocument)
```

Comment:
It's difficult to test the caching, because there's no span for us to set it as an attribute (which is how we test the validation-cache). Should I add a span for that, or should we expose it as a header, like we do with the Execution Plan Cache? 

https://github.com/wundergraph/cosmo/blob/06fd0e93bdeff2e9cecf7e39cf9db5c8e219a8df/router/core/graphql_handler.go#L39

Reply:
Either is fine